hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -911,48 +911,50 @@ public void receives_one_clearEvent_after_mapClear_call_from_client() {
         IMap<Object, Object> anotherClientMap = anotherClient.getMap(clientMap.getName());
         anotherClientMap.clear();
 
-        // sleep for a while to see there is another clear event coming
-        sleepSeconds(2);
-
         assertTrueEventually(new AssertTask() {
```

Comment:
Two assertions makes it hard to aware what is being tested. We can introduce 2 test instead if two assertions are needed. 

Also you are right, `assertTrueAllTheTime` was confused me.